docs/book/beyond_basics/webhook_installer_generator.md

```diff
@@ -0,0 +1,139 @@
+# Webhook Installer vs Webhook Manifests Generator
+
+There are 2 options to install webhook configurations into a cluster:
```

Comment:
I realized it will make sense to add one or two sentences here to explain the main motivation. For ex. installing webhook config requires higher permissions which mgr's service account might not have. In that case, a process with higher permissions would like to install webhook configuration.
And then dive in to two approaches.

Reply:
Done